src/functions/__tests__/getDependentsGraph.test.js

```diff
@@ -6,7 +6,7 @@ import fixtures from 'fixturez';
 const f = fixtures(__dirname);
 
 describe('function/getDependentsGraph', () => {
-  describe('A project with nested workspaces', async () => {
```

Comment:
Broke with updated jest flow typed file. Async not required as far as I can tell